analyzer/codechecker_analyzer/analyzer_context.py

```diff
@@ -269,10 +315,29 @@ def checkers_severity_map_file(self):
         return os.path.join(self._package_root, 'config',
                             'checker_severity_map.json')
 
+    @property
+    def checkers_guideline_map_file(self):
+        # Get coding guideline map file from the environment.
+        if 'CC_GUIDELINE_MAP_FILE' in os.environ:
```

Comment:
Please extend the help message of the analyze/check commands with this new environment variable. There is a section for environment variables at the bottom of the help message. Please don't forget to update the docs too.